libraries/Microsoft.Bot.Builder.ApplicationInsights/Microsoft.Bot.Builder.ApplicationInsights.csproj

```diff
@@ -7,6 +7,8 @@
     <PackageVersion Condition=" '$(IsBuildServer)' != '' ">$(ReleasePackageVersion)</PackageVersion>
     <Configurations>Debug;Release</Configurations>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <DocumentationFile>bin\$(Configuration)\netstandard2.0\Microsoft.Bot.Builder.ApplicationInsights.xml</DocumentationFile>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
   </PropertyGroup>
 
   <PropertyGroup>
```

Comment:
Do we need to suppress documentation warnings for this project?

Reply:
No, all of the documentation is there, any new changes that don't have docs will be treated as errors going forward, makes sense?

Reply:
Perfect sense. Thanks.

---
In reply to: [439581296](https://github.com/microsoft/botbuilder-dotnet/pull/4061#discussion_r439581296) [](ancestors = 439581296)